app-backend/db/src/main/resources/migrations/V34__update_make_grid_function.sql

```diff
@@ -0,0 +1,60 @@
+CREATE OR REPLACE FUNCTION public.ST_MakeGrid (
+  bound_polygon public.geometry,
+  width_step double precision,
```

Comment:
the only change here is to take doubles instead of ints, since with task pixel size and the COG's resolution we don't know we'll get an int out